complete/contract-rest-service/src/main/java/hello/PersonService.java

```diff
@@ -0,0 +1,24 @@
+package hello;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+
+import org.springframework.stereotype.Service;
+
+@Service
+class PersonService {
+
+	private Map<Long, Person> personMap = new HashMap<>();
+	
+	@PostConstruct
+	public void init() {
```

Comment:
Actually we can move this to a constructor instead of calling `@PostConstruct`, right?

Reply:
fixed. thanks